<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="glWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>-20</y>
      <width>961</width>
      <height>891</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>158</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>-1</number>
     </property>
     <item>
      <widget class="QPushButton" name="up_btn">
       <property name="text">
        <string> ↑</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="left_btn">
         <property name="text">
          <string>←</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down_btn">
         <property name="text">
          <string>↓</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="right_btn">
         <property name="text">
          <string>→</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>104</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="z_down_btn">
       <property name="text">
        <string> ↑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="z_up_btn">
       <property name="text">
        <string>↓</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>148</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Количество полигонов:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="number_of_facets">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>109</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Толщина вершин</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="points_widthSpinBox">
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>20.000000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="background_color">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>580</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Цвет фона</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбор Проекции</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>131</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Скорость движения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="move_spinBox">
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>131</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Количество вершин:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="number_of_verts">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>202</width>
      <height>118</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="rot_ox_pl_btn">
         <property name="text">
          <string> ↪ X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rot_ox_min_btn">
         <property name="text">
          <string>X ↩</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="rot_oy_pl_btn">
         <property name="text">
          <string>↪ Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rot_oy_min_btn">
         <property name="text">
          <string>Y ↩</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="rot_oz_min_btn">
         <property name="text">
          <string>↪  Z</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rot_oz_pl_btn">
         <property name="text">
          <string> Z ↩</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>99</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Толщина линей</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="lines_widthSpinBox">
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>20.000000000000000</double>
       </property>
       <property name="value">
        <double>8.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>121</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QPushButton" name="parallel_projec_btn">
       <property name="text">
        <string>Параллельная</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="centre_projec_btn">
       <property name="text">
        <string>Центральная</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Задать масштаб</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scale_spinBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>0.900000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>61</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Угол</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_angle">
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="value">
        <number>7</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>187</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Движение по Z</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1222</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuLines">
    <property name="title">
     <string>Lines</string>
    </property>
    <widget class="QMenu" name="menuType">
     <property name="title">
      <string>Type</string>
     </property>
     <addaction name="actionDashed"/>
     <addaction name="actionSolid"/>
    </widget>
    <addaction name="menuType"/>
    <addaction name="actionLineColor"/>
   </widget>
   <widget class="QMenu" name="menuPointer">
    <property name="title">
     <string>Pointer</string>
    </property>
    <widget class="QMenu" name="menuType_2">
     <property name="title">
      <string>Type</string>
     </property>
     <addaction name="actionNone"/>
     <addaction name="actionSquare"/>
     <addaction name="actionSmooth"/>
    </widget>
    <addaction name="menuType_2"/>
    <addaction name="actionColorPointer"/>
   </widget>
   <widget class="QMenu" name="menuImg_gif">
    <property name="title">
     <string>Img/gif</string>
    </property>
    <addaction name="actionMake_img"/>
    <addaction name="actionMake_gif"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLines"/>
   <addaction name="menuPointer"/>
   <addaction name="menuImg_gif"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSwitch_color">
   <property name="text">
    <string>Switch color</string>
   </property>
  </action>
  <action name="actionDashed">
   <property name="text">
    <string>Dashed</string>
   </property>
  </action>
  <action name="actionSolid">
   <property name="text">
    <string>Solid</string>
   </property>
  </action>
  <action name="actionLineColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionNone">
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionSquare">
   <property name="text">
    <string>Square</string>
   </property>
  </action>
  <action name="actionSmooth">
   <property name="text">
    <string>Smooth</string>
   </property>
  </action>
  <action name="actionColorPointer">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionMake_img">
   <property name="text">
    <string>Make img</string>
   </property>
  </action>
  <action name="actionMake_gif">
   <property name="text">
    <string>Make gif</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glWidget</class>
   <extends>QWidget</extends>
   <header location="global">glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
